config/linux_gpu.cmake

```diff
@@ -24,7 +24,8 @@
 #
 #  $ cp config/linux_gpu.cmake config.cmake
 #
-#  Next modify the according entries, and then compile by
+#  Next modify the entries in the config.cmake like MXNET_CUDA_ARCH to set the specific
+#  GPU architecture, and then compile by
```

Comment:
Thanks @leezu for that feedback, how about this where we point out that users might want to set the MXNET_CUDA_ARCH when using the linux_gpu.cmake file. And then let them refer below for the specific details. This at least points out that they might need to do something with MXNET_CUDA_ARCH in order to build for GPU